packages/tc-schema-sdk/lib/updater.js

```diff
@@ -27,7 +27,14 @@ class SchemaUpdater {
 		if (schemaDirectory && !schemaData) {
 			schemaData = {
 				schema: {
-					types: readYaml.directory(schemaDirectory, 'types'),
+					types: [].concat(
+						readYaml.directory(schemaDirectory, 'types'),
+						readYaml.directory(schemaDirectory, 'relationships'),
```

Comment:
Only 1 question from me now. Why is it needed to include relationships in here. Would prefer to keep types 100% separate from relationships in these rawData lists. What are you using this combined list for?

Reply:
Ah, I followed your comment: https://github.com/Financial-Times/treecreeper/pull/69#discussion_r352167849 but maybe I misunderstand. this is not necessary, so I'll remove this line